Makefile

```diff
@@ -18,24 +18,40 @@
 # NOTE: git-version-gen will generate a value for VERSION, unless you override it.
 IMAGE_TAG_BASE ?= ghcr.io/nearnodeflash/nnf-mfu
 
+# CONTAINER_TOOL defines the container tool to be used for building images.
+# Be aware that the target commands are only tested with Docker which is
+# scaffolded by default. However, you might want to replace it to use other
+# tools. (i.e. podman)
+CONTAINER_TOOL ?= docker
+
+# Which tag to pull from the cray lustre source code.
+# **Note:** This must also be set in .github/workflows/main.yaml for both
+# production and debug docker builds.
+CRAY_LUSTRE_VERSION ?= cray-2.15.B19
+
+# Only support linux/amd64 builds. This reduces complexity with installing
+# lustre, which is an option enabled for mpifileutils.
+PLATFORM ?= linux/amd64
```

Comment:
Does this mean we can't build a Mac version for use in Kind anymore?

Reply:
I believe I can drop this explicit `PLATFORM` definition and with the recent additions that I've made, we can build for arm as well. I'll test this out more.